kolibri/plugins/coach/assets/src/views/common/QuizHeader.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+
+  <KPageContainer style="padding-top:16px;">
+    <BackLink
+      :to="backlink"
+      :text="backlinkLabel"
+    />
+
+    <!-- Cheating to get the same layout effect but not
+         using a backlink...
+    -->
+    <BackLinkWithOptions>
```

Comment:
Agreed but it might be easier to do it in https://github.com/learningequality/kolibri/pull/5996 (which branches from this branch) where I continue to refactor the same component similarly in Lessons - just a little bit easier to do it in one sweep on that branch than to do it here and fix it there during rebase?

Reply:
that's fine!